package.json

```diff
@@ -79,8 +79,11 @@
     "fuse-concat-with-sourcemaps": "^1.0.5",
     "glob": "^7.1.2",
     "ieee754": "^1.1.11",
+    "inquirer": "^3.0.6",
     "lego-api": "^1.0.8",
+    "minimist": "^1.2.0",
     "mustache": "^2.3.0",
+    "node-sass": "^4.9.0",
     "postcss": "^6.0.22",
```

Comment:
Node sass might have slipped by mistake

Reply:
@nchanged wtf?
@orblazer Please.... squash your commits & follow `git-cz` rules for scope, etc...

Reply:
Yes I have resolve the conflit from my phone and on github we can't resolve conflit with custom commit message :(

Reply:
@nchanged What you mean by "Node sass might have slipped by mistake" ?

Reply:
@orblazer so?